eng/testing/tests.wasm.targets

```diff
@@ -22,14 +23,27 @@
     <BuildAOTTestsOn Condition="'$(BuildAOTTestsOn)' == ''">local</BuildAOTTestsOn>
   </PropertyGroup>
 
+  <!-- 
+    This is running during compile time and therefore $(Scenario) is empty, unless the specific project sets it.
+    Any settings in the project file could be replaced on Helix.
+    See also eng\testing\WasmRunnerTemplate.sh
+    -->
+  <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
+    <SetScriptCommands Condition="'$(Scenario)' != ''" Include="export SCENARIO=$(Scenario)" />
+    <SetScriptCommands Condition="'$(JSEngine)' != ''" Include="export JS_ENGINE=--engine=$(JSEngine)" />
+    <SetScriptCommands Condition="'$(JSEngineArgs)' != ''" Include="export JS_ENGINE_ARGS=$(JSEngineArgs)" />
+    <SetScriptCommands Condition="'$(_WasmMainJSFileName)' != ''" Include="export MAIN_JS=--js-file=$(_WasmMainJSFileName)" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <SetScriptCommands Condition="'$(Scenario)' != ''" Include="set &quot;SCENARIO=$(Scenario)&quot;" />
+    <SetScriptCommands Condition="'$(JSEngine)' != ''" Include="set &quot;JS_ENGINE=--engine=$(JSEngine)&quot;" />
+    <SetScriptCommands Condition="'$(JSEngineArgs)' != ''" Include="set &quot;JS_ENGINE_ARGS=$(JSEngineArgs)&quot;" />
+    <SetScriptCommands Condition="'$(_WasmMainJSFileName)' != ''" Include="set &quot;MAIN_JS=--js-file=$(_WasmMainJSFileName)&quot;" />
+  </ItemGroup>
   <PropertyGroup Condition="'$(RunScriptCommand)' == ''">
-    <JSEngine Condition="'$(JSEngine)' == ''">V8</JSEngine>
-    <JSEngineArgs Condition="'$(JSEngine)' == 'V8'">$(JSEngineArgs) --engine-arg=--stack-trace-limit=1000</JSEngineArgs>
-
```

Comment:
Where are these set now?

Reply:
They could be set explicitly by individual .csproj, if they are not, the default value would be calculated in the RunScript on the Helix. See https://github.com/dotnet/runtime/pull/62779/files#diff-5cb02ee3bbdcb7cb7e772808b494689a7c5a83e1539645bfef2003ac78fc7702R37